src/test/kotlin/three/consulting/epoc/service/TimesheetServiceIntegrationTest.kt

```diff
@@ -139,4 +140,16 @@ class TimesheetServiceIntegrationTest : IntegrationTest() {
             .isInstanceOf(UnableToDeleteTimesheetException::class.java)
             .hasMessage("Cannot delete timesheet, no timesheet found for given id: 1000")
     }
+
+    @Test
+    fun `searching timesheet with project id 1 returns array of timesheet objects`() {
+        val timesheet: List<TimesheetDTO>? = timesheetService.findTimesheetForProjectId(1L)
+        assertThat(timesheet).hasSize(2)
+    }
+
+    @Test
+    fun `searching timesheet with project id 99 returns empty array`() {
+        val timesheet: List<TimesheetDTO>? = timesheetService.findTimesheetForProjectId(99L)
+        assertThat(timesheet).hasSize(0)
+    }
```

Comment:
Naming issues here too. You're managing collections not single objects so timesheet -> timesheets